containers/src/Kokkos_ScatterView.hpp

```diff
@@ -558,6 +558,8 @@ struct ReduceDuplicatesBase {
       Kokkos::Profiling::beginParallelFor(std::string("reduce_") + name, 0,
                                           &kpID);
     }
+#else
+    (void)name;
```

Comment:
Apparently, we never tried to compile without profiling in any of the other regression testers.